src/WebJobs.Script/FileProvisioning/FuncAppFileProvisioningService.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
nit: Can remove unused using statements.

Reply:
The focus of this was on the mechanical changes to replace the duplicate constant. I'll apply some of the suggestions (they're valid), but try to keep the PR focused on the original intent.